src/components/Annotations/Circle.tsx

```diff
@@ -3,28 +3,26 @@ import { SVGProps } from 'react';
 import { useEllipsePosition } from '../../hooks';
 
 export interface AnnotationCircleProps
-  extends Omit<
-    SVGProps<SVGCircleElement>,
-    'x1' | 'x2' | 'y1' | 'y2' | 'cx' | 'cy' | 'r'
-  > {
-  cx: number | string;
-  cy: number | string;
+  extends Omit<SVGProps<SVGCircleElement>, 'cx' | 'cy' | 'r'> {
+  x: number | string;
+  y: number | string;
   r: number | string;
+  color?: string;
 }
 
 export function Circle(props: AnnotationCircleProps) {
-  const { cx: oldCx, cy: oldCy, r: oldR, ...otherProps } = props;
+  const { x: oldX, y: oldY, r: oldR, color, ...otherProps } = props;
 
   const {
-    cx,
-    cy,
+    x,
+    y,
```

Comment:
can you make the hook still return `cx` and `cy`, as that is the name of the true props

Reply:
okey fixed